packages/codegen/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "private": true,
   "name": "@cognite/sdk-codegen",
+  "version": "0.0.0-development",
```

Comment:
why did you add version?

Reply:
Rather: can you verify that this won't start publishing the codegen package on changes?

Reply:
The commit message should explain. Yarn complains when packages does not contain a version (it's required for a package.json file)

Reply:
Version is only required if you plan to publish a package. I don't believe that's a goal here.

Reply:
Seems like a bug in yarn then, would make more sense to do a PR to yarn instead of adjusting our codebase. I really don't think we should do releases for this package, which I'd argue is more significant than a unnecessary warning.

Reply:
`"private": true` will prevent Lerna/NPM from publishing the package. See https://github.com/lerna/lerna/tree/main/commands/publish#usage and https://docs.npmjs.com/cli/v8/configuring-npm/package-json#private

We have `--no-private` in the `lerna version` command (see `release.yml` workflow), so this package should not be changed by that neither.

Reply:
I've tested this against a local registry. The version and publish command does not touch the codegen package at all, so this should be fine. I can't comment on that issue, but this might be because we are using more flags and specific variants of the commands.

```
Successfully published:
 - @cognite/sdk-beta@5.1.2
 - @cognite/sdk-core@4.4.1
 - @cognite/sdk-playground@5.3.2
 - @cognite/sdk@7.6.2
lerna success published 4 packages
```